src/current/molt/molt-fetch.md

```diff
@@ -597,6 +599,79 @@ By default, MOLT Fetch moves all data from the [`--source`](#source-and-target-d
 --table-filter '.*user.*' --table-exclusion-filter '.*temp.*'
 ~~~
 
+### Selective data movement
+
+Use `--filter-path` to specify the path to a JSON file that defines row-level filtering for data load. This enables you to move a subset of data in a table, rather than all data in the table. To apply row-level filters during replication, you need a [userscript](#filter-path-userscript-for-replication).
+
+{% include_cached copy-clipboard.html %}
+~~~
+--filter-path 'data-filter.json'
+~~~
+
+The JSON file should contain one or more entries in `filters`, each with a `resource_specifier` (`schema` and `table`) and a SQL expression `expr`. For example, the following example exports only rows from `public.t1` where `v > 100`:
+
```

Comment:
Is it worth showing the example rows that get matched? I remember Jane had some testing she did that shows the rows written to the source, and the ones that end up on the target after filters are applied.

Reply:
https://github.com/cockroachdb/molt/blob/main/e2e/testdata/pg/filter_ddt.ddt#L71-L107
Could use the DDT output as example?

We dont show the exact source data, but 
```
exec
psql 'postgres://postgres@localhost:{pgport}/e2edb?sslmode=disable' -c "INSERT INTO filtertbl1 SELECT * FROM generate_series(1, 30);"
----
INSERT 0 30

exec
psql 'postgres://postgres@localhost:{pgport}/e2edb?sslmode=disable' -c "SELECT COUNT(*) FROM filtertbl;"
----
count 
-------
    30
(1 row)
```
should be pretty self explanatory.